server/app.js

```diff
@@ -44,13 +46,12 @@ app.use("/users", auth, usersRouter);
 app.use("/login", loginRouter);
 app.use("/spotify", spotifyRouter);
 app.use("/posts", auth, postsRouter);
+app.use("/likes", auth, postLikes);
+app.use("/commentsLike", auth, commentLikes);
 app.use("/comments", auth, commentRouter);
 app.use("/profile", profileRouter);
 app.use("/message", messageRouter);
-app.use("/conversation", conversationRouter);
-app.use("/follow", followRouter);
-
```

Comment:
We need the followRouter

Reply:
Since this last version doesn't have the following functionality ready. I've removed it for now. We will implement it later after the demo, don't worry about that.